indexer/index.cpp

```diff
@@ -112,6 +113,29 @@ bool Index::FeaturesLoaderGuard::IsWorld() const
   return m_handle.GetValue<MwmValue>()->GetHeader().GetType() == feature::DataHeader::world;
 }
 
+unique_ptr<FeatureType> Index::FeaturesLoaderGuard::GetOriginalFeatureByIndex(uint32_t index) const
+{
+  auto feature = make_unique<FeatureType>();
+  if (!GetOriginalFeatureByIndex(index, *feature))
+    return unique_ptr<FeatureType>();
+
+  return feature;
+}
+
+unique_ptr<FeatureType> Index::FeaturesLoaderGuard::GetOriginalOrEditedFeatureByIndex(uint32_t index) const
+{
+  auto feature = make_unique<FeatureType>();
+  if (!m_handle.IsAlive())
```

Comment:
If handle is not alive, why not to return nullptr?

Reply:
ok

Reply:
```c++
return {};
```